distributed/scheduler.py

```diff
@@ -3003,6 +3004,27 @@ def __init__(
             distributed.dashboard.scheduler.connect(
                 self.http_application, self.http_server, self, prefix=http_prefix
             )
+        self.jupyter = jupyter
+        if self.jupyter:
+            from jupyter_server.serverapp import ServerApp
+            from traitlets.config import Config
+
+            j = ServerApp.instance(
+                config=Config(
+                    {
+                        "ServerApp": {
+                            "base_url": "jupyter",
+                            "token": "",
```

Comment:
As with #6407, this is a big security hole. I'm not worried about it for an off-by-default experiment, but flagging for later if we want to start promoting this.

Reply:
If you can see and connect to this server then you can likely already run arbitrary code on it.

This isn't always true (it actually isn't true for Coiled today) but I'd argue that that's more the fault of whoever is presenting insecure Dask dashboards.  It's also opt-in.